MAUI/Circular-Charts/DoughnutChart.md

```diff
@@ -103,4 +103,58 @@ chart.Series.Add(series);
 
 {% endtabs %}
 
-![Semi doughnut chart in MAUI Chart](Chart-Types_images/maui_semi_doughnut_chart.png)
\ No newline at end of file
+![Semi doughnut chart in MAUI Chart](Chart-Types_images/maui_semi_doughnut_chart.png)
+
+## Center View
+
+The view placed in the center of the doughnut chart is useful for sharing additional information about the doughnut chart. Any view can be added to the center of the doughnut chart using the [CenterView]() property of [DoughnutSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.DoughnutSeries.html). The binding context of the [CenterView]() will be the respective DoughnutSeries.
+
+### Center Hole Size
+
+The [CenterHoleSize]() property of [DoughnutSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.DoughnutSeries.html) is used to get the diameter value of the doughnut center hole. Using this [CenterHoleSize]() value, you can provide [CentreView]() for the series to prevent the view from overlapping the series.
+
```

Comment:
Using this [CenterHoleSize]() value, you can provide [CentreView]() for the series to prevent the view from overlapping the series.

@BaraniBharathi18, change like below. 
Using the CenterHoleSize, we can protect the view in the doughnut center from overlapping with the series.

Reply:
Complete the changes and update PR